tests/unit/proxy/test_junos.py

```diff
@@ -8,16 +9,14 @@
 from tests.support.unit import TestCase, skipIf
 
 if sys.version < "3":
-    from six import StringIO
+    from salt.ext.six import StringIO
```

Comment:
FYI - Support of Python 2 is no longer required, hence additions for six are no longer required and support for using six shall be removed at a later date.

Reply:
fixed via https://github.com/saltstack/salt/pull/57281/commits/87dda5eddd9593b67e1bb98caea2cb56e2bf2980